### PR TITLE
feat: add mapbox controls

### PR DESCRIPTION
**Pull request by raphael-arce**
added mapbox controls and overwrote mapbox styles

### CODE REVIEW

File: src/components/map-attribution/map-attribution.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    i18n.map.attribution.improve,
+    i18n.map.attribution.feedback,
+    i18n.map.attribution.imprint,
+    i18n.map.attribution.privacy,
+  ];
+
+  return (
+    <p
+      className={`
+      pointer-events-auto absolute bottom-[4.3rem] right-1 max-w-60
```
Comment (raphael-arce) [inline src/components/map-attribution/map-attribution.tsx:19]: @Jaszkowic I did it with absolute here bc I wasn't sure how to make it with grid. Could you help me with that?

File: src/components/map/map.tsx [hunk 2]
```diff
@@ -9,7 +10,14 @@ const Map: React.FC = () => {
 
   useMapSetup(mapContainer);
 
-  return <div ref={mapContainer} className="h-screen" />;
+  return (
+    <div className="grid grid-cols-1 grid-rows-1">
+      <div className="z-[1000] col-start-1 row-start-1 flex items-end justify-end">
```
Comment (aeschi) [inline src/components/map/map.tsx:15]: this div stretches across the entire map and blocks the user from interacting with the map (no zoom, moving around etc)

**Review by aeschi — CHANGES_REQUESTED** [review]
- the zoom in and out works well when blending out the map attributions 👍 
- when I click on locate, I get located BUT the locate button dissapears :D

- there is currently some white space under the map making the page bigger than the window height:
Screenshot 2024-03-07 at 16 31 07

**Comment by raphael-arce** [comment]
@aeschi I think I fixed everything, and I can't reproduce the dissapearing locate me button, could you look into it again?

**Comment by raphael-arce** [comment]
@aeschi i think i fixed it now. could you please test again?

**Review by aeschi — APPROVED** [review]
LGTM

**Pull request merged**